index.js

```diff
@@ -22,6 +22,15 @@ function municipios(departamento){
   }
 }
 
+function poblacion(departamento){
+  if (departamento == null) {
+    return data.poblacionDeptoObj;
+  } else {
+    const ldep = repDep(departamento);
+    return data.poblacionDepto[ldep];
+  }
+}
+
```

Comment:
Add a jsDoc Comment to the function.
So developers can check what the function fo.
``` js
/** # Title
 *  info and example
 */
function poblacion(departamento){
  if (departamento == null) {
    return data.poblacionDeptoObj;
  } else {
    const ldep = repDep(departamento);
    return data.poblacionDepto[ldep];
  }
}
```

Reply:
Done

Reply:
Solve